packages/react-components/react-combobox/src/components/Listbox/useListbox.ts

```diff
@@ -48,6 +49,24 @@ export const useListbox_unstable = (props: ListboxProps, ref: React.Ref<HTMLElem
     matchOption: el => el.classList.contains(optionClassNames.root),
   });
 
+  const onActiveDescendantChange = useListboxContext_unstable(ctx => ctx.onActiveDescendantChange);
+
+  const listenerRef = React.useCallback(
+    (el: HTMLDivElement | null) => {
+      if (!el) {
+        return;
+      }
+
+      el.addEventListener('activedescendantchange', (untypedEvent: Event) => {
+        // Typescript doesn't support custom event types on handler
+        const event = untypedEvent as ActiveDescendantChangeEvent;
+        event.stopImmediatePropagation();
```

Comment:
not sure about this one - better would be to peroperly remove event listener.
without it Dropdown catches the events multiple times